### ISSUE TITLE
loss is Nan

### ISSUE THREAD

**Issue by rookiecoder-chen** [materialsvirtuallab/megnet#123]
I use all features for training, but at the beginning, loss is Nan. What's the reason? Is there a solution?
`structures = [mol_from_file(molpath + filename + '.mol', file_format='mol') for filename in filenames]
targets = df['rou(exp)'].values

#split data
train_structures = structures[:1500]
test_structures = structures[1500:]
train_targets = targets[:1500]
test_targets = targets[1500:]

#build model
mg = MolecularGraph(atom_features=['element', 'chirality', 'formal_charge', 'ring_sizes', 
                                    'hybridization', 'donor', 'acceptor', 'aromatic'],
                                    bond_features=['bond_type', 'same_ring', 'spatial_distance', 'graph_distance'],
                                    distance_converter=GaussianDistance(np.linspace(0, 4, 20), 0.5),
                                    known_elements=['C', 'H', 'N', 'O'])
model = MEGNetModel(27, 2, 32, nblocks=3, lr=1e-20, n1=64, n2=64, n3=64, embedding_dim=64,npass=3, ntarget=1, graph_converter=mg)

INTENSIVE = True # U0 is an extensive quantity
scaler = StandardScaler.from_training_data(train_structures, train_targets, is_intensive=INTENSIVE)
model.target_scaler = scaler

model.train(train_structures, train_targets, batch_size=1,epochs=50, verbose=2,
            callbacks=[ReduceLRUponNan()], scrub_failed_structures=False, prev_model=None,
            lr_scaling_factor=0.5, patience=500,)  # In reality, use epochs>1000`

**Comment by chc273** [comment]
It maybe caused by nan values or very large abs errors initially.

1. Check your target and see if there is any nan. 2. Make sure the target value is small. You may need to transform your target values, e.g. standardize it or divide it by some values.

**Comment by rookiecoder-chen** [comment]
My target is float numbers between 1 and 2,

INTENSIVE = True # U0 is an extensive quantity
scaler = StandardScaler.from_training_data(train_structures, train_targets, is_intensive=INTENSIVE)
model.target_scaler = scaler

Isn't this part of the code doing the scaler?By the way, there is no problem when I use megnet simple model. The problem only occurs in megnet full model. Can the notenook of transfer learning give a complete demo?

**Comment by chc273** [comment]
Without knowing details about your target values and no reproducible codes, I cannot comment much. You may need to scale your input features before putting them to model. Transfer learning example is complete. You just need to train the model the way you train other models after setting the transferred model weights.
